draft-ietf-acme-acme.md

```diff
@@ -1916,8 +1924,14 @@ It MUST NOT contain any characters outside the URL-safe Base64 alphabet and MUST
 NOT contain any padding characters ("=").
 
 ~~~~~~~~~~
+GET /acme/authz/1234/0 HTTP/1.1
```

Comment:
The same is true of all the other example sections, where you added the obscuring HTTP/JWS glue. I happen to agree that the glue is distracting, but we should be consistent about including or not including it.